components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -618,7 +591,6 @@ void RegisterPrefsForBraveReferralsService(PrefRegistrySimple* registry) {
   registry->RegisterIntegerPref(kReferralAttemptCount, 0);
 #if BUILDFLAG(IS_ANDROID)
   registry->RegisterTimePref(kReferralAndroidFirstRunTimestamp, base::Time());
-  registry->RegisterStringPref(kSafetynetStatus, std::string());
```

Comment:
whenever deprecates prefs, we need to clear its existing value from preferences.
You can refer `kNewTabPageShowGemini` how it's cleared after its usage is deleted.